templates/page_header/page_header.component.inc

```diff
@@ -13,6 +13,11 @@ function europa_atomium_theme_page_header() {
     'page_header' => array(
       'variables' => array(
         'markup' => NULL,
+        'header_bottom' => NULL,
+        'site_name' => NULL,
+        'page_title' => NULL,
+        'page_intro' => NULL,
+        'page_meta' => NULL,
       ),
```

Comment:
I think it would be best to keep the names of the variables as close to the ones on the ECL as possible, that will improve making the relationship between both.